.github/workflows/security-check.yaml

```diff
@@ -0,0 +1,52 @@
+name: Security Checks
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+env:
+  FAIL_LEVEL: WARNING
+  ENV_TYPE: pipenv
+  DEP_PATH: src/
+  APP_PATH: src/
+  EXTRA_ARGS: "--settings=registrar.config.settings"
+  DJANGO_SECRET_KEY: not-a-secret-jw7kQcb35fcDRIKp7K4fqZBmVvb+Sy4nkAGf44DxHi6EJl
+  DATABASE_URL: "postgres://not_a_user:not_a_password@not_a_host"
+  DJANGO_BASE_URL: "https://not_a_host"
```

Comment:
Comment vars, please. I spent a while trying to figure out why OWASP wasn't failing on warnings before realizing that `FAIL_LEVEL` was intended for victoriadrake/django-security-check

Reply:
Re-arranged so that these env vars were under the particular job they were needed for.